### PR TITLE
handle postgres date type when dates are BC

### PR DESCRIPTION
**Pull request by jnewmano**
Handle the case when a postgres column of date type has date values that are BC.  For example, 0011-02-03 BC.

### CODE REVIEW

File: encode_test.go [hunk 1]
```diff
@@ -100,10 +118,23 @@ func TestParseTs(t *testing.T) {
...
+				i, tt.str, tt.timeval, val)
+		}
+	}
+}
+
 // Now test that sending the value into the database and parsing it back
 // returns the same time.Time value.
 func TestEncodeAndParseTs(t *testing.T) {
-	db, err := openTestConnConninfo("timezone='Etc/UTC'")
+	db, err := openTestConnConninfo("")
```
Comment (johto) [inline encode_test.go]: Why this change?

**Comment by jnewmano** [comment]
I have updated the pull request to get rid of the unnecessary code change that you pointed out.
